website/content/en/pre-docs/tasks/running-pods.md

```diff
@@ -0,0 +1,222 @@
+---
+title: "Running pods"
+linkTitle: "Running pods"
+weight: 10
+---
+
+If your pods have no requirements for how or where to run, you can let Karpenter choose nodes from the full range of available cloud provider resources.
+However, by taking advantage of Karpenter's model of layered constraints, you can be sure that the precise type and amount of resources needed are available to your pods.
+Reasons for constraining where your pods run could include:
+
+* Saving money by running in ways that are more efficient
+* Needing to run in zones where dependent applications or storage are available
+* Requiring certain kinds of processors or other hardware
+* Wanting to use techniques like topology spread to help insure high availability
+
+Your Kubernetes cluster operator adds the first layer of constraints by creating one or more provisioners.
+The next layer comes from you adding specifications to your Kubernetes pod deployments.
+When you request constraints, those constraints must fall within the provisioner constraints or the pods will not deploy.
+For example, if the provisioner sets limits that allow only a particular zone to be used, and a pod asks for a different zone, deployment will fail.
+
+Constraints you can request include:
+
+* **Resource requests**: Request that certain amount of memory or CPU be available.
+* **Disruption budget**: Limit the disruptions that an application can experience.
+* **Node selection**: Choose to run on a node that is has a particular label (`nodeSelector`) or name (`nodeName`).
+* **Topology spread**: Use topology spread to help insure availability of the application.
+* **Node affinity**: Draws a pod to run on nodes with particular attributes (affinity) or that don't have those attributes (antiaffinity).
+* **Persistent volumes**: Insures the availability of selected persistent volumes.
+
+The constraints that Karpenter supports are based mostly on features built into Kubernetes and a few that are specific to the cloud provider.
+In fact, you the exact same constraints can be used on clusters by Cluster Autoscaler or even just by the Kubernetes scheduler to pre-provision capacity.
+Keep in mind, however, that not all Kubernetes constraints are supported or recommended with Karpenter.
+This section steps through examples of those that are supported.
```

Comment:
I want to be really careful that we don't communicate that karpenter lacks support for critical scheduling constraints.
```suggestion
```